analyzer/consensus/consensus.go

```diff
@@ -293,11 +293,12 @@ func (m *Main) processBlock(ctx context.Context, height int64) error {
 	}
 
 	for _, f := range []func(*storage.QueryBatch, *storage.StakingData) error{
-		m.queueTransfers,
+		m.queueRegularTransfers,
 		m.queueBurns,
 		m.queueEscrows,
 		m.queueAllowanceChanges,
 		m.queueStakingEvents,
```

Comment:
does anything weird happen in here?

Reply:
No, this just inserts (= logs) the events that come from the `staking` app (_and_ are not associated with a tx) as-is into the `events` table.
We do have a similar, more aptly (IMO) named fn `queueTxEventInserts`. I went ahead and renamed this one in the same vein, to `queueStakingEventInserts`, so it sounds less like there's some processing of events going on.

I also renamed another 4 or 5 similarly named fns (one per tendermint app) in the same way.